plugin/sampling/strategystore/static/strategy_store_test.go

```diff
@@ -79,7 +78,7 @@ func TestPerOperationSamplingStrategies(t *testing.T) {
 	os := s.OperationSampling
 	assert.EqualValues(t, os.DefaultSamplingProbability, 0.8)
 	require.Len(t, os.PerOperationStrategies, 4)
-	fmt.Println(os)
+	//fmt.Println(os)
```

Comment:
remove altogether

Reply:
Thank you for suggestions, I will remove it in the later revision.